dangerfile.js

```diff
@@ -79,3 +79,59 @@ if (!isDependabotPR) {
     // Check Reviewer 2
     checkReviewerSection(reviewer2Section ? reviewer2Section[0] : null, 'Reviewer 2');
 }
+
+// README checks for Web Components
+const readmeFiles = [...danger.git.created_files, ...danger.git.modified_files]
```

Comment:
Not sure how I feel about doing these checks, at least at the danger level. IMO it already creates quite a lot of noise.

Wouldn't this be solved by just updating the generator README template? Then once we've refactored README's for existing components, there shouldn't be a need for this?

Reply:
As discussed offline - will keep this for now and have added exclusions for pie-webc, pie-webc-core and pie-webc-testing. Will raise a follow up PR to update the component generator too so that all new components adhere to this structure. Cheers Ben!